examples/antd/index.html

```diff
@@ -15,11 +14,6 @@
 
 <body>
     <div id="example-app"></div>
-    <script
-        src="https://code.jquery.com/jquery-3.1.1.min.js"
-        integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
-        crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/antd/3.7.1/antd.js"></script>
```

Comment:
This was the problem: adding `antd` twice: once by this script and a second time by the script generated by parcel...